src/content/docs/faq.mdx

```diff
@@ -250,11 +250,24 @@ although the base of that path is provided by the Android system and can change
 dynamically at runtime. On iOS devices, data is stored under
 `Library/Application Support/shorebird`.
 
+Shorebird stores only a few files, one is `state.json` which contains the state
+information for the updater. When patches are installed there is also a "slots"
+directory containing up to two directories (0, 1) for each potential downloaded
+patch. These will contain the inflated copies of the patches, which will be
```

Comment:
This bit about slots is no longer true. Patches are now stored in their own directory, and we should never have more than two on disk at the same time. Once we've successfully booted from patch N+1, patch N's artifacts are deleted.